packages/rolldown/tests/fixtures/output/footer/_config.ts

```diff
@@ -0,0 +1,29 @@
+import type {
+  RollupOptions,
+  RolldownOutput,
+  RolldownOutputChunk,
+} from '../../../../src'
+import { expect } from 'vitest'
```

Comment:
This should be `import { ... } from 'rolldown'` directly. And you don't need to import these types with using `defineTest`. Types will be inferred automatically.

Reply:
Ok, I will fix these .

Reply:
Nice. I create an [issue](https://github.com/rolldown/rolldown/issues/695) for this. Not just the banner test, some other tests also have this issue.

Reply:
Ok